packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -17,10 +17,10 @@
   --terra-alert-title-float: left;
   --terra-alert-title-margin-bottom: 0;
   --terra-alert-title-margin-right: 0.5rem;
-  --terra-alert-actions-margin-bottom: 1rem;
+  --terra-alert-actions-margin-bottom: 0.6158rem;
   --terra-alert-actions-margin-left: 0;
   --terra-alert-actions-margin-right: 0.5rem;
-  --terra-alert-actions-margin-top: 1rem;
+  --terra-alert-actions-margin-top: 0.6158rem;
```

Comment:
What i am seeing is 0.658rem makes the alert height equivalent to default theme. can you test that one again ? also cross check once in clinical theme whether the same issue exists.

Reply:
In default theme the alert height with button is 42px and without button is 41px.
In fusion theme I have made both the alerts to be 41px. Should I make all variants as 41px or keep 41px without button and 42px with button.
cc @jeremyfuksa

Reply:
If everything can be 41px, then we should aim for consistency.

Reply:
In that case, I will update default theme with button to be 41px also

Reply:
Updated in 378a40a